src/main/java/nextstep/subway/section/domain/Distance.java

```diff
@@ -20,12 +24,31 @@ public Distance(int distance) {
 		this.distance = distance;
 	}
 
-	public int getDistance() {
-		return this.distance;
+	private void validate(int distance) {
+		if (MIN_DISTANCE >= distance)
+			throw new InvalidDistanceException("거리는 0보다 작을 수 없습니다.");
 	}
 
-	private void validate(int distance) {
-		if(MIN_DISTANCE >= distance)
-			throw new IllegalArgumentException("거리는 0보다 작을 수 없습니다.");
+	public Distance getDifferenceDistance(Distance distance) {
+		int differenceDistance = this.distance - distance.distance;
+		if (differenceDistance < 0) {
+			throw new InvalidDistanceException("새로운 역을 등록할 경우 기존 역 사이 길이보다 크거나 같으면 등록할 수 없습니다.");
+		}
```

Comment:
피드백 반영 잘해주셨네요 👍
이부분도 검증 메소드로 분리해보면 더 좋겠네요 :)

```suggestion
		if (differenceDistance < MIN_DISTANCE) {
			throw new InvalidDistanceException("새로운 역을 등록할 경우 기존 역 사이 길이보다 크거나 같으면 등록할 수 없습니다.");
		}
```